app/views/users/index.html.erb

```diff
@@ -7,5 +7,14 @@
 
   <body>
     <h1> This is the Users page</h1>
+    <% @users.each do |user| %>
+      <a href="/users/<%= user.id%>">
+        <div></div>
+        <div>
+          <h2><%= user.name %></h2> 
+          <span>Number Of Posts: <%= user.posts_counter %></span>
+        </div>
+      </a>
```

Comment:
- [optional ] Please add a default value of zero to each counter so we don't get this error 
![image](https://user-images.githubusercontent.com/74744787/216770270-20921401-1036-450e-a20a-7f3c3182fb7a.png)

Reply:
Thank you for your feedback, due to the deadline I am skipping the optional tasks but I will surely act upon them on coming Sunday.